File: TWBAndroid/app/src/main/java/twb/conwaybrian/com/twbandroid/model/Article.java
package twb.conwaybrian.com.twbandroid.model;


import java.util.ArrayList;
import java.util.Date;
import java.util.List;

import lombok.Data;

@Data
public class Article {
    private String articleId;
    private String content;
    private List<String> images;
    private String userId;
    private String title;
    private int points;
    private int views;
    private int commentCount;
    private Date createTime;
    //private LocalDateTime createTime;
    private int likeStatus;

    public Article() {
        images = new ArrayList<>();
    }
}

File: TWB-ios/TWB-ios/model/Article.swift
//
//  Article.swift
//  TWB-ios
//
//  Created by Brian on 2019/5/13.
//  Copyright © 2019 Brian. All rights reserved.
//

import Foundation
import Alamofire
struct  Article {
     var articleId:String
     var content:String
     var images:[String]
     var userId:String
     var title:String
     var points:Int64
     var views:Int64
     var likeStatus:Int64
     var createTime:Date
    
    func toJson() ->  Parameters{
        return [:]
    }
}
